src/blockdata/block.rs

```diff
@@ -231,17 +231,27 @@ impl Block {
         bitcoin_merkle_root(hashes).into()
     }
 
+    /// The size of the header + the size of the varint with the tx count + the txs themselves
+    #[inline]
+    fn get_base_size(&self) -> usize {
+        80 + VarInt(self.txdata.len() as u64).len()
+    }
+
     /// Get the size of the block
     pub fn get_size(&self) -> usize {
-        // The size of the header + the size of the varint with the tx count + the txs themselves
-        let base_size = 80 + VarInt(self.txdata.len() as u64).len();
         let txs_size: usize = self.txdata.iter().map(Transaction::get_size).sum();
-        base_size + txs_size
+        self.get_base_size() + txs_size
+    }
+
+    /// Get the strippedsize of the block
+    pub fn get_strippedsize(&self) -> usize {
```

Comment:
```suggestion
    pub fn get_stripped_size(&self) -> usize {
```

just to be consistent with other function names?